README.md

```diff
@@ -1578,6 +1579,36 @@ The [Eio_unix][] module provides features for using Eio with OCaml's Unix module
 In particular, `Eio_unix.run_in_systhread` can be used to run a blocking operation in a separate systhread,
 allowing it to be used within Eio without blocking the whole domain.
 
+### Domainslib
+
+For certain compute-intensive tasks it may be useful to send work to a pool of [Domainslib][] worker domains.
+You can resolve an Eio promise from non-Eio domains (or systhreads), which provides an easy way to retrieve the result.
+For example:
+
+<!-- $MDX skip -->
+```ocaml
+let pool = Domainslib.Task.setup_pool ~num_domains:2 ()
+
+let fib n = ... (* Some Domainslib function *)
```

Comment:
Do you think it makes sense to stress that you _cannot_ perform a call that may perform an (Eio) effect in the Domainslib async task because that may be run in a different domain that no longer has the effect handler installed?

Reply:
Good idea. I've added a bit of text saying you can mostly only use functions from the appropriate library in each domain.